datadog_lambda/trigger.py

```diff
@@ -114,10 +114,14 @@ def parse_event_source(event: dict) -> _EventSource:
 
     event_source = None
 
-    request_context = event.get("requestContext")
+    # Get requestContext safely and ensure it's a dictionary
+    request_context = event.get("requestContext", {})
```

Comment:
it's better to do `request_context = event.get("requestContext") or {}` becuase it will avoid an allocation in the case that the key is found in the dict.

Reply:
that's some expert level advice on allocation, thanks!